### ISSUE TITLE
Item is wider than widget

### ISSUE THREAD

**Issue by greenfrvr** [greenfrvr/hashtag-view#4]
There is an unhandled case. When the value of measured text width + left/right paddings + left/right drawables is greater than total widget width, an infinite loop appears on items sorting step.

**Comment by lawloretienne** [comment]
I'm not sure why, but the version in your README does not match the version available here
http://gradleplease.appspot.com/#hashtag-view

**Comment by greenfrvr** [comment]
are you using mavenCentral? sorry, that's was my fault, I forgot to sync bintray repo with mavenCentral, now last version should be available.

**Comment by lawloretienne** [comment]
I'm using jCenter() which I think is the preferred repo over mavenCentral().  I just checked again.  I think it takes a few hours to go live before it shows up on http://gradleplease.appspot.com/#hashtag-view

**Comment by lawloretienne** [comment]
I just tested this again in v1.1.1 and it still is an issue.

**Comment by greenfrvr** [comment]
Can't really get where is the problem. I have tested widget with long tags and it worked fine. Did you try to use `tagMaxWidth` property?

**Comment by lawloretienne** [comment]
Okay I figured out the issue, I added paddingLeft and paddingRight to the HashtagView style and this is what caused the garbage collection to happen.

``` xml
<style name="HashtagView">
        <item name="android:layout_width">match_parent</item>
        <item name="android:layout_height">wrap_content</item>
        <item name="android:paddingLeft">16dp</item> <!-- Changed -->
        <item name="android:paddingRight">16dp</item> <!-- Changed -->
        <item name="android:paddingTop">16dp</item> <!-- Changed -->
        <item name="tagTextSize">16dp</item> <!-- Changed -->
</style>
```

**Comment by greenfrvr** [comment]
finally I got the problem, fixed in v1.2.0
